examples/wordpress/README.md

```diff
@@ -0,0 +1,160 @@
+# Wordpress Example
+
+Welcome to the Wordpress example. It shows you how to deploy a Wordpress application using Deployment Manager.
+
+## Prerequisites
+
+
+### Deployment Manager
+First, make sure DM is installed in your Kubernetes cluster by following the instructions in the top level
+[README.md](../../README.md).
+
+### Google Cloud Resources
+The Wordpress application will make use of several persistent disks, which we will host on Google Cloud. To create these disks we will create a deployment using Google Cloud Deployment Manager:
+```gcloud deployment-manager deployments create wordpress-resources --config wordpress-resources.yaml```
+
+where `wordpress-resources.yaml` looks as follows:
+
+```
+resources:
+- name: nfs-disk
+  type: compute.v1.disk
+  properties:
+    zone: us-central1-b
+    sizeGb: 200
+- name: mysql-disk
+  type: compute.v1.disk
+  properties:
+    zone: us-central1-b
+    sizeGb: 200
+```
+
+### Privileged containers
+To use NFS we need to be able to launch privileged containers. If your Kubernetes cluster doesn't support this you can either manually change this in every Kubernetes minion or you could set the flag at `kubernetes/saltbase/pillar/privilege.sls` to true and (re)launch your Kubernetes cluster. Once Kubernetes 1.1 releases this should be enabled by default.
+
+### NFS Library
+Currently the nfs-common library should be installed by default on Kubernetes nodes. In case nfs-common is not installed you can install it on all Kubernetes nodes using the following command:
+```
+gcloud compute instances list | cut -d ' ' -f 1 | tail -n +2 | xargs -n1 gcloud compute ssh --command="sudo apt-get update;sudo apt-get -y install nfs-common"
```

Comment:
If you're going to give `gcloud` commands, then `nfs-common` is installed (now), unless they happen to be bringing up CoreOS nodes. I ensured it before 1.1 shipped.